### ISSUE TITLE
[Android Bootcamp] Project 0: Todo - Ready for Review

### ISSUE THREAD

**Issue by hearthanah** [hearthanah/SimpleTodo#8]
My app is complete, please review. /cc @codepathreview @codepath

I used a Basic Activity instead of a Blank Activity like the tutorial instructed. I believe Android Studio has recently changed these options, there is no longer Blank Activities. What is the difference? 

I also noticed my android app, by default, has a different color scheme than the tutorials. (mine has a blue title bar, while the tutorial is black or dark gray). Is this due to different versions or is this a custom setting thing?

Thanks! :]

**Comment by nesquena** [comment]
Hi Hanah,

Looks good, and thanks for submitting your pre-work. We hope you enjoyed working on the project! Unfortunately, due to high demand this round, we are restricting participation in July to those with at least 1 year of experience after graduating with an engineering degree (excluding internships). We hope you are able to participate in a future cohort though!

In the meantime, there are a lot of resources out there for people interested in learning Android and we’ve done some work to make them more accessible. We [have many online resources for learning Android](http://courses.codepath.com/snippets/intro_to_android/email_faq#heading-i-do-not-live-in-the-bay-area-can-i-take-your-classes-locally-or-remotely) including a long list of [Android resources online](http://guides.codepath.com/android/Beginning-Android-Resources#beginning-android-resources). You can also follow [us on Twitter @codepath](https://twitter.com/codepath) for news of future courses.

Hope to be able to expand our audience for future programs. Thanks for your understanding,

**Comment by nesquena** [comment]
> I used a Basic Activity instead of a Blank Activity like the tutorial instructed. I believe Android Studio has recently changed these options, there is no longer Blank Activities. What is the difference?

You are right, they renamed Blank to Empty. Basic contains a bunch of extra code and has two layout files per activity: `activity_main.xml` which contains the top toolbar and scrolling container code and `content_main.xml` where you put your screen contents. Empty (blank) activity, has just one file  `activity_main.xml` which was empty and where you put your screen contents. There's several other differences as well all where basic has extra code included by default.

> I also noticed my android app, by default, has a different color scheme than the tutorials. (mine has a blue title bar, while the tutorial is black or dark gray). Is this due to different versions or is this a custom setting thing?

Yes, good catch! Recently they changed the top bar to have a different default color to match their new style guidelines.
